src/transformers/models/fsmt/configuration_fsmt.py

```diff
@@ -125,7 +125,10 @@ class FSMTConfig(PretrainedConfig):
     ```python
     >>> from transformers import FSMTConfig, FSMTModel
 
+    >>> # Initializing a FSMT configuration
```

Comment:
Hi @sha016 
Thank you for the PR.

For this model, I think we should remove `"facebook/wmt19-en-ru"` below, and just use `FSMTConfig()`.
You can follow what has been done here for `bert`

https://github.com/huggingface/transformers/blob/98c9c5add92aa66d9f09af192139b4cb502aa9fb/src/transformers/models/bert/configuration_bert.py#L128

Let me know if this would work 🙏